api/Makefile

```diff
@@ -3,7 +3,7 @@
 .PHONY: lock run stop test
 # Pin the dependencies
 lock:
-	pip install poetry>=1.0
+	pip install poetry>=1.0 poetry-plugin-export
```

Comment:
@odulcy-mindee That's required because `poetry-plugin-export` will be dropped from future poetry releases